setup.py

```diff
@@ -3,7 +3,7 @@
 setup(
     name='reportportal-client',
     packages=find_packages(),
-    version='3.0.0',
+    version='3.1.0',
```

Comment:
Are you sure about this changes does not break backward compability and work similarly for 3.x and 4.x Report Portal API?

Reply:
I'll do the tests and return with some answers

Reply:
I tested the code on both RP 3 and RP 4.
The 'parameters' argument is simply being ignored in RP 3

Reply:
But do you want me to update the version to 3.2.0?
There's already 3.1.0 on PyPI

Reply:
The correct way might be `3.0.1` and than `3.1.0`.

To be honest, this is bugfix, thus according to `semver` it should be 3.2.0 (as you mentioned).
Also, please, do not forget to update `download_url='https://github.com/reportportal/client-Python/tarball/3.0.0',` to right download url.

Reply:
Nice catch. Fixed